kore-json.py

```diff
@@ -25,7 +25,7 @@ def print_kast(data, sort="SortJSON"):
     sys.stdout.write(')')
   elif isinstance(data, OrderedDict):
     sys.stdout.write("Lbl'LBraUndsRBraUnds'EVM-DATA'UndsUnds'JSONList{}(")
-    for key, value in data.items():
+    for key, value in list(data.items()):
```

Comment:
Why is it necessary to collect dictionary items to a list before iteration?

Reply:
Not sure, I changed it back.